docker-app/qfieldcloud/core/migrations/0073_project_packaging_offliner.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.23 on 2024-01-09 22:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0072_person_remaining_trial_organizations"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="project",
+            name="packaging_offliner",
+            field=models.CharField(
+                choices=[
+                    ("qgiscore", "QGIS Core Offline Editing (deprecated)"),
+                    ("pythonmini", "Optimized Packager"),
+                ],
+                default="qgiscore",
+                help_text='The packaging offliner is is used to package data for offline use with QField. The new "Optimized Packager" should be preferred over the deprecated "QGIS Core Offline Editing" for newer projects.',
```

Comment:
```suggestion
                help_text='The `packaging_offliner` packages data for offline use with QField. The new "Optimized Packager" should be preferred over the deprecated "QGIS Core Offline Editing" for new projects.',
```

Reply:
```suggestion
                help_text='The Packaging Offliner packages data for offline use with QField. The new "Optimized Packager" should be preferred over the deprecated "QGIS Core Offline Editing" for new projects.',
```

We should not use developer names for the UI